code/__HELPERS/game.dm

```diff
@@ -391,7 +391,7 @@
 			var/mob/living/carbon/human/H
 			if(ishuman(M.current))
 				H = M.current
-			return M.current.stat != DEAD && !issilicon(M.current) && !isbrain(M.current) && (!H || H.dna.species.id != "memezombies")
+			return M.current.stat != DEAD && !issilicon(M.current) && !isbrain(M.current) && (!H || H.dna.species.id != "memezombies" || H.dna.species.id != "memezombiesfast")
```

Comment:
H.dna.species.id != "memezombies" || H.dna.species.id != "memezombiesfast"
This will always be true